braindecode/training/scoring.py

```diff
@@ -273,8 +283,9 @@ def on_epoch_end(self, net, dataset_train, dataset_valid, **kwargs):
             for cb in epoch_cbs:
                 cb.y_preds_ = y_preds
                 cb.y_trues_ = y_test
-
         # y pred should be same as self.y_preds_
+        # robintibor@gmail.com: Unclear if this also leads to any
```

Comment:
you have git blame for this. I would not clutter the code with email adresses

Reply:
ok nagut, removed